### DISCUSSION TITLE
When running the training script I get the error `Illegal Instruct (core dumped)` how to resolve?

### DISCUSSION THREAD

**Discussion by Haseeeb21** [PaddlePaddle/PaddleOCR#14069] [Q&A]
I am following the steps mentioned in https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/detection_en.md#21-start-training
I have added my custom dataset path in the `det_mv3_db.yml` file, and also downloaded the pre-trained `MobileNetV3_
large_x1_0_pretrained.pdparams` and then
I am running the following command,
`python3 tools/train.py -c configs/det/det_mv3_db.yml` and getting the error 
*Illegal Instruct (core dumped)*

Help me fix this so that I can train PaddleOCR

**Comment by GreatV** [comment]
Hi @Haseeeb21, you might need to install Paddle 3.0. To update, visit this URL: https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/develop/install/pip/linux-pip.html

**Reply by GreatV** [reply]
Refer to the Chinese installation document, as the English version is outdated.

**Reply by Haseeeb21** [reply]
> Refer to the Chinese installation document, as the English version is outdated.

can you provide link? which one should I follow
- https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/installation.md
- https://github.com/PaddlePaddle/PaddleOCR/blob/main/test_tipc/docs/install.md
- https://github.com/PaddlePaddle/PaddleOCR/blob/main/docs/ppocr/installation.md


and is it necessary to do with docker? as I have my custom dataset and using MobileNetV3_large_x1_0_pretrained model can you guide me,
which python version is supported? does 3.10 works?

**Reply by GreatV** [reply]
you may need to install Paddle 3.0. visit this URL: https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/develop/install/pip/linux-pip.html

**Reply by Haseeeb21** [reply]
> you may need to install Paddle 3.0. visit this URL: https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/develop/install/pip/linux-pip.html

I have done this now to train the model on custom dataset which documentation should I follow?

**Reply by GreatV** [reply]
please refer to: https://paddlepaddle.github.io/PaddleOCR/latest/ppocr/model_train/training.html
